borgmatic/config/schema.yaml

```diff
@@ -1609,6 +1609,238 @@ properties:
                 example:
                     - start
                     - finish
+    pushover:
+        type: object
+        additionalProperties: false
+        properties:
+            token:
+                type: string
+                description: |
+                    Your application's API token.
+                example: 7ms6TXHpTokTou2P6x4SodDeentHRa
+            user:
+                type: string
+                description: |
+                    Your user/group key (or that of your target user), viewable 
+                    when logged into your dashboard: often referred to as 
+                    USER_KEY in Pushover documentation and code examples.
+                example: hwRwoWsXMBWwgrSecfa9EfPey55WSN
+            start:
+                type: object
+                type: object
+                properties:
+                    message:
+                        type: string
+                        description: |
+                            Message to be sent to the user or group.
+                        example: A backup job has started.
+                    priority:
+                        type: integer
+                        description: |
+                            A value of -2, -1, 0 (default), 1 or 2 that indicates the message priority.
+                        example: 0
+                    expire:
+                        type: integer
+                        description: |
+                            he expire parameter specifies how many seconds 
+                            your notification will continue to be retried 
+                            for (every retry seconds).
+                        example: 1200
+                    retry:
+                        type: integer
+                        description: |
+                            The retry parameter specifies how often 
+                            (in seconds) the Pushover servers will send the 
+                            same notification to the user.
+                        example: 30
+                    device:
+                        type: string
+                        description: |
+                            The name of one of your devices to send just to that device instead of all devices.
+                        example: pixel8
+                    html:
+                        type: integer
```

Comment:
I can do this however it means that I need to loop through my state_config to convert the `True` to a 1. Is that ok?

```python
state_config = {k: (1 if v is True else v) for k, v in state_config.items()}
```

Reply:
That sounds like it would work. (Although please expand [single-letter variables](https://torsion.org/borgmatic/docs/how-to/develop-on-borgmatic/#code-style).) But here's a Python trick that would make this a little easier!

```python
state_config = {key: int(value) for key, value in state_config.items()}
```

But, I hear you saying, that'll break on every field *other* than the boolean `html`! Okay, no problem:

```python
state_config = {key: int(value) for key if key == `html` else value, value in state_config.items()}
```

And then if you wanted it more declarative, you could even do it without modifying `state_config`. Example:

```python
    data = dict(
        {
            'token': token,
            'user': user,
            'message': state.name.lower(),  # default to state name. Can be overwritten in state_config loop below.
        },
        **{key: int(value) for key if key == `html` else value, value in state_config.items()},
    )
```

Do not feel strongly. Just giving you some options.

Reply:
Done. Just needed some slight modifications but it's passing all tests: https://github.com/borgmatic-collective/borgmatic/pull/86/commits/61e4eeff6ca9b7b0659960a306ba87879832b62a

Reply:
Thanks, but if I'm reading it correctly, I don't think this it handles the case where the `html` option is False. Doesn't that need to get converted to a `0`? That's why I was suggesting the `int()` trick.. It converts `True` to `1` and `False` to `0` so you don't have to implement that logic yourself!

Reply:
Ah, I think I finally got it: https://github.com/borgmatic-collective/borgmatic/pull/86/commits/5321301708186ea6733295e1279a8a68b105b0c5

Reply:
This should do it: https://github.com/borgmatic-collective/borgmatic/pull/86/commits/5f3d4f9b03b6832781ebf071952a37a9afecfb97

Reply:
Looks good!